### ISSUE TITLE
Link types settings of field shows up only after we initially save the field

### ISSUE THREAD

**Issue by piotrpog** [verbb/hyper#95]
### Describe the bug

Link types settings of field shows up only after we initially save the field.

### Steps to reproduce

1. create new field, select field type hyper
2. Field type setting widget does not show up - only setting label "Field type"
3. after saving the field, widget shows up.

### Craft CMS version

4.5.6

### Plugin version

1.1.13

### Multi-site?

No

**Comment by engram-design** [comment]
This one is a Craft issue, [fixed here](https://github.com/craftcms/cms/commit/1a958a7c9e85568f060004e369ca8d02f020a105)

**Comment by stevendierick** [comment]
Actually, this might still be an issue as I'm experiencing something similar but with nested Hyper fields via slideout/settings panel:

https://share.cleanshot.com/5bHnZXc5mb2T7jPDkzTM

Nested Hyper fields (via slideout) only work if you save right after adding links on the parent level, and thus only after each save when adding a new parent link. If you add a new parent link without saving first then adding nested Hyper links won't work. It throws a JavaScript error `Cannot set properties of undefined (setting 'your_field_handle')`.

Also, when adding and saving nested Hyper links (via slideout) and re-open the settings panel without saving the entry, it won't show your nested Hyper links anymore. BUT if you cancel the slideout and then save the entry, the nested Hyper links will have been added. BUT if you would have saved the slideout, with the empty nested Hyper links, and continue to save your page then the nested Hyper links will have been removed (or still empty).

Running `dev-craft-4 as 1.1.13` and Craft CMS 4.5.6.1.

**Comment by engram-design** [comment]
This issue is regarding the **field settings**, so when creating a new field, changing the type to Hyper, the Link Types Vue component to configure your link types didn't appear. This would be a separate issue.

Should be fixed for the next release. To get this early run `composer require verbb/hyper:"dev-craft-4 as 1.1.13"`

**Comment by engram-design** [comment]
Fixed in [1.1.14](https://github.com/verbb/hyper/releases/tag/1.1.14)
